modules/serverless-sinkbinding-kn.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+//
+// serverless/knstive_eventing/serverless-sinkbinding.adoc
+
+[id="serverless-sinkbinding-kn_{context}"]
+= Using SinkBinding with the Knative CLI (`kn`)
+
+ This guide describes the steps required to create, manage and delete a SInkBinding instance using `kn` commands.
+
+.Prerequisites
+
+* You will need to have a Knative Serving and Eventing installation.
+* You will need to have the `kn` CLI installed.
+
+.Procedure
+
+. To check that SinkBinding is set up correctly, create a Knative event display service, or event sink, that dumps incoming messages to its log.
++
+You can create the service by entering the following command:
++
+----
+$ kn service create event-display --image quay.io/openshift-knative/knative-eventing-sources-event-display
+----
+
+. Create a SinkBinding that will direct events to the service.
++
+You can create the SinkBinding by entering the following command:
++
+----
+$ kn source sinkbinding create testevents --sink broker:default --resource "event:v1" --service-account events-sa
```

Comment:
The options are not correct (they are different than for the api server source). Actually a SinkBinding example is a bit more involved as you also need the other end for the binding (one is the broker). This can be a `Deployment`, `Job`, `StatefulSet` or anything which carries a Pod template and eventually fire up a Pod.

I would suggest to use a `Deployment` but the problem here, I don't know if we have a sample image out of the box that could be used here. Because the application within the image needs to honor the `K_SINK` environment variable which will get injected by Knative Operator to point to the broker's URL so that the application can send CloudEvents to the broker.

Let me check whether there is an example already available upstream. If not, I'm afraid we would need to create first such an application and make it generally available (like the `knative-eventing-sources-eventing-display` image mentioned above.